src/fios.cpp

```diff
@@ -397,7 +397,7 @@ static void FiosGetFileList(SaveLoadOperation fop, fios_getlist_callback_proc *c
 	{
 		SortingBits order = _savegame_sort_order;
 		_savegame_sort_order = SORT_BY_NAME | SORT_ASCENDING;
-		QSortT(file_list.files.Begin(), file_list.files.Length(), CompareFiosItems);
+		QSortT(file_list.files.data(), file_list.Length(), CompareFiosItems);
```

Comment:
It's a bit more work involved, but since you're converting some stuff in this PR other than just converting SmallVector, perhaps converting to use std::sort would be a good idea as well

Alternatively, it should be a separate PR (along with the various other changes in this PR) :)

Reply:
I agree that the standard algorithms would be ideal where applicable, a seperate PR would make mose sense.

Reply:
Here an untested suggestion: (at least `#include<algorithm>` is missing)
```suggestion
		std::sort(file_list.files.data(), file_list.files.data() + file_list.Length(), CompareFiosItems);
```